app/src/main/java/com/epfl/drawyourpath/utils/Utils.kt

```diff
@@ -264,4 +247,128 @@ object Utils {
         val y = ln(tan(lat) + (1 / cos(lat)))
         return Point.create(long.toFloat(), y.toFloat())
     }
+
+    /**
+     * Gets the smallest coordinates independently.
+     * When we want to fit the path in a square, it represents the topleft corner of the square.
+     * @return The smallest coordinate for all the strokes.
+     */
+    fun getSmallestPoint(strokes: List<Stroke>): Point =
+        strokes.fold(Pair(Float.MAX_VALUE, Float.MAX_VALUE)) { acc, stroke ->
+            getSmallestPoint(stroke).let {
+                Pair(min(acc.first, it.x), min(acc.second, it.y))
+            }
+        }.let { Point.create(it.first, it.second) }
+
+    /**
+     * Gets the smallest coordinates independently.
+     * When we want to fit the path in a square, it represents the topleft corner of the square.
+     * @return The smallest coordinate for the stroke.
+     */
+    fun getSmallestPoint(stroke: Stroke): Point =
+        stroke.points.fold(Pair(Float.MAX_VALUE, Float.MAX_VALUE)) {
+                acc, point ->
+            Pair(min(acc.first, point.x), min(acc.second, point.y))
+        }.let { Point.create(it.first, it.second) }
```

Comment:
These two functions are quite similar even if one uses the other. One alternative I can think of is : 
```kotlin
fun getSmallestPoint(strokes: List<Stroke>): Point =
        strokes.map { it.points }.flatten().fold(Pair(Float.MAX_VALUE, Float.MAX_VALUE)) { acc, point ->
            Pair(min(acc.first, point.x), min(acc.second, point.y))
        }.let { Point.create(it.first, it.second) }
```
and the other
```kotlin
fun getSmallestPoint(stroke: Stroke): Point = getSmallestPoint(listOf(stroke))
```